### PR TITLE
Copy open source required doc files from branch publication/nbS1-2023 into main

### PR DESCRIPTION
**Pull request by WeinaJi**
This PR is aimed for making the `main` branch as default.
* Copy doc/license/banner files
* Copy and enable the workflow script on main

### CODE REVIEW

**Review by jorblancoa — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -44,7 +49,7 @@ Build special with mod files
   mkdir mods
   cp -r mod-files-from-released-circuit mods/
   cp -r neurodamus-install-prefix/share/mod/* mods/
-  nrnivmodl -incflags '-I <include-paths-of-our-dependencies>' -loadflags '-L <libs-paths-for-linking>' mod
+  nrnivmodl -incflags '-I <include-paths-of-our-dependencies>' -loadflags '-L <libs-paths-for-linking>'' mod
```
Comment (ferdonline) [inline README.rst:52]: Too many closing quotation marks?

**Review by ferdonline — DISMISSED** [review]
Thanks. Noticed only possible typo. Also, I thought we would include a copyright to some mod files, did I miss them?

**Comment by WeinaJi** [comment]
> Also, I thought we would include a copyright to some mod files, did I miss them?

Yes, it is also planned in https://bbpteam.epfl.ch/project/issues/browse/BBPBGLIB-1047. But I don't know which files need a additional copy right. Do you know?

File: README.rst (hunk 1 above)
Comment (WeinaJi) [reply README.rst:52]: ah, yes, good spot. I will remove it.

**Review by jorblancoa — APPROVED** [review]

**Comment by ferdonline** [comment]
> But I don't know which files need a additional copy right. Do you know?

I suspect it's only `VecStim.mod` and `netstim_inhpoisson.mod` from core.mod since they are the only extra ones. If that makes sense to you, maybe we can just the notice and close the topic. Later we can improve as you say

**Review by ferdonline — APPROVED** [review]

**Pull request merged**